Modules/_functoolsmodule.c

```diff
@@ -145,7 +145,7 @@ typedef struct {
 } partialobject;
 
 // cast a PyObject pointer PTR to a partialobject pointer (no type checks)
-#define _PyPartialObject_CAST(PTR)  ((partialobject *)(PTR))
+#define partialobject_CAST(op)  ((partialobject *)(op))
```

Comment:
There' no need to change *existing* macros, especially ones with the `_Py` prefix that's de-facto reserved for Python.

Reply:
Err actually this is something I probably wrote recently so I thought it would have been better to make it consistent at least. I can revert it though (I think that's my code because there is a PTR and this is something I used before Victor told me to use "op")

Reply:
Oh, right, it's from #124733 from October